### PR TITLE
Upgrade to httptransfer v2.6.0

### PR DESCRIPTION
**Pull request by bobvanmanen**
## Description

* Error notification and handling for assets without rejecting the entire asset list
* Logging improvements
* Support for asset versioning (when the asset name conflicts during upload)

## How Has This Been Tested?

* Unit testing
* Testing using a manual app written running in Node v6.10.3

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/direct-binary-upload-process.js [hunk 1]
```diff
@@ -81,6 +83,19 @@ export default class DirectBinaryUploadProcess extends FileTransferHandler {
...
+            httpTransferLogger.debug.log = (...args) => log.debug.apply(log, args);
+            httpTransferLogger.info.log = (...args) => log.info.apply(log, args);
+            httpTransferLogger.warn.log = (...args) => log.warn.apply(log, args);
+            httpTransferLogger.error.log = (...args) => log.error.apply(log, args);
+            // debug.enable("httptransfer:*");
+        }
+
+        httpTransferLogger.info("test info");
+        httpTransferLogger.warn("test warn");
+        httpTransferLogger.error("test error");
```
Comment (mfrisbey) [inline src/direct-binary-upload-process.js:98]: Are these "debugging" messages that should be removed?

**Review by mfrisbey — COMMENTED** [review]
Just one question about log messages.

File: src/direct-binary-upload-process.js (hunk 1 above)
Comment (bobvanmanen) [reply src/direct-binary-upload-process.js]: I've removed them

**Review by mfrisbey — APPROVED** [review]
Thanks!

**Pull request merged**